src/test/java/com/google/graphgeckos/dashboard/github/GitHubControllerTests.java

```diff
@@ -1,4 +1,4 @@
-package com.google.graphgeckos.dashboard;
+package com.google.graphgeckos.dashboard.github;
 
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
```

Comment:
(suggestion for another PR): we've missed this wildcard import; please expand it.